packages/gluestick-env-expose-ast/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "jest",
     "test-coverage": "jest --coverage",
-    "lint": "eslint ./",
+    "lint": "eslint ./*.js ./__tests__/**/*.js",
```

Comment:
Use `eslint .` and then create a `.eslintignore` that ignores all `/coverage` folders.